lib/assessment.js

```diff
@@ -39,7 +71,15 @@ var newArray2 = function newArray2(length, defaultsFunction) {
 // The function you write should return
 // the object passed in as an argument or the object created
 var addMethod = function addMethod(propertyName, method, obj) {
-
+  // not completely sure how to approach
+  if (!obj) {
+    return {};
+  }
+  else {
+    function(obj) {
+      propertyName.method = propertyName.method;
+    }
+  }
```

Comment:
Clarifying the question may help here. This method is supposed to take an object and add a method to it. But what if no object is passed in? In that case, we should make a new object, and add the method to that.

As a set of steps, that becomes:
1. Check to see if `obj` is undefined or not; if it is, set `obj` to an empty object.
2. Add new property (propertyName: method) to `obj`
3. Return `obj`

As code, that could be written
```javascript
if (!obj) { obj = {}; }
obj[propertyName] = method;
return obj;
```